packages/components/src/InputFile/InputFile.tsx

```diff
@@ -207,6 +234,9 @@ export function InputFile({
               {hintText}
             </Typography>
           )}
+          {fileRejections?.length > 0 && (
+            <div className={styles.validationErrors}>{validationErrors}</div>
```

Comment:
curious what would happen here if `validationErrors` contained multiple elements

is it going to have a problem with not having a single fragment/parent?

Reply:
hmm so the layout is a bit iffy, I don't love how this shows up

![image](https://github.com/GetJobber/atlantis/assets/11843215/2a5af303-cdd8-4e87-ae6e-3b29b9196680)

also since we're doing

```
  const validationErrors = fileRejections?.map(({ file, errors }) => {
    return errors.map(error => {
      return (
        <InputValidation
          message={file.name + "\n" + error.message}
          key={file.name}
        />
      );
```

we get the multiple identical key console error

```
react-dom.development.js:86 Warning: Encountered two children with the same key, `Additional_Tools_for_Xcode_14.1.dmg`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

Reply:
![image](https://github.com/GetJobber/atlantis/assets/13531657/a2d77c08-7bb6-4ca5-bc72-225d8d11bb8e)
I think it should be fine. This is an example of what can happen

Reply:
I think it would look cleaner if it had `flex-direction:column; justify-content:flex-start`

![image](https://github.com/GetJobber/atlantis/assets/13531657/6f4fca0e-c73f-4c6e-ac17-ed3d98b299ad)

Reply:
right but in your case you don't have two violations for a single file which is not that farfetched if we have say a type limit, a size limit and a quantity limit

you'd hit that non unique key problem

I'm curious why yours are stacking and mine aren't. is that just a screen size thing? if so, I think the one I ended up with is pretty not great/not that readable and we should do something to ensure they always stack.

Reply:
I'm updating the css class to be 
```
.validationErrors {
  display: flex;
  justify-content: center;
  flex-direction: column;
}
```

Reply:
>> you'd hit that non unique key problem
I can update the key to be `{file.name}{error.code}` that should fix the unique issue